mod-source-record-manager-server/src/main/java/org/folio/services/MappingRuleService.java

```diff
@@ -14,17 +15,18 @@
    * Returns rules in JsonObject
    *
    * @param tenantId tenant
+   * @param recordType type of rules (MARC_BIB or MARK_HOLDING)
    * @return optional with rules in JsonObject
    */
-  Future<Optional<JsonObject>> get(String tenantId);
+  Future<Optional<JsonObject>> get(String tenantId, String recordType);
```

Comment:
Will be better to have `tenantId` as last parameter. Please check this in all updated methods.